src/main/java/frc/robot/commands/TrackTarget.kt

```diff
@@ -1,72 +1,176 @@
 package frc.robot.commands
 
+import com.github.chen0040.glm.solvers.Coefficients
```

Comment:
Nit: we should probably define the coefficients type ourselves now that we've moved away from this library. This has the additional benefit of allowing us to give names for each of the three coefficients rather than numbers (which are somewhat confusingly currently 2 minus the order).